src/settings.js

```diff
@@ -98,15 +98,15 @@ var INVOKE_RUN = 1;
 //  - For a command (has a main function) this is always 1
 //  - For a reactor (no a main function) this is always 0
 // [link]
-var EXIT_RUNTIME = 0;
+var EXIT_RUNTIME = false;
```

Comment:
Reading the settings.js file now, it does feel a bit awkward that the options now read `false` or `true`, since that is not what users are expected to set the values to.

My understanding is that users commonly look into this file to check for the documentation and default values of settings, as there is a lot of documentation and info in this file. (at least I do that all the time)

Now what can then happen is that users see some setting being false, and then they can easily read "I'll set -sEXIT_RUNTIME=true` to make it work.

Though not sure how big of an issue that will be in practice.

Reply:
I see what you mean yes.   The good thing is that now that we have type checking anyone to tries to use `=true` on the command line will get a nice error message.

A possible mitigation would be to:

(a) Document this different at the top of the file.
(b) Support `=true` / `=false` on the command like as alternatives to `=1/=0`

In the long run however, I think we should convert this file to something other than JS.   It makes no sense that this is a JS file today since the only place we read it is now in python :)